can/interfaces/canalystii.py

```diff
@@ -66,7 +66,9 @@ class VCI_CAN_OBJ(Structure):
 
 
 class CANalystIIBus(BusABC):
-    def __init__(self, channel, device=0, baud=None, Timing0=None, Timing1=None, can_filters=None):
+    def __init__(
+        self, channel, device=0, baud=None, Timing0=None, Timing1=None, can_filters=None, **kwargs
```

Comment:
I think this wasn't backported correctly? I believe this is missing part of the change from 934a753671d243729e92c9a045bb344e0dab6ee8

```suggestion
        self, channel, device=0, bitrate=None, Timing0=None, Timing1=None, can_filters=None, **kwargs
```

Reply:
Also we're missing the docstring change to go along with it

Reply:
I figured it best to avoid changing the API? So I purposely avoided changing baud to bitrate for a backported bugfix

Reply:
Hmm, perhaps we want to consider supporting both this version, with `baud` having a warning that it'll be removed? I believe it's the only interface using `baud` instead of `bitrate`.

Reply:
That is fair. I'll push something now.